xbmc/platform/darwin/osx/Info.plist.in

```diff
@@ -38,5 +38,7 @@
 	<string>Translate speech to text in virtual keyboard dialog</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>Used for speech recognition</string>
+	<key>NSAppleEventsUsageDescription</key>
+	<string>Used for intercepting media keys</string>
```

Comment:
is it needed? I didn't see any new permission requests...

Reply:
Yes, it's requested when the application first opens to let you allow kodi into the usability permissions.